_layouts/default.html

```diff
@@ -8,6 +8,7 @@
     <meta name="theme-color" content="#157878">
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    <link rel="icon" href="{{ site.baseurl }}/favicon.ico" type="image/x-icon">
```

Comment:
1. We should use the `absolute_url` or `relative_url` filters, rather than appending the baseurl directly
2. Is `type="image/x-icon"` necessary? What does it do?
3. What happens if the user doesn't have a favicon? Should we wrap it in a conditional?

Reply:
3. I suggested the usage of a conditional in #32 and I think that would be the optimal configuration.

> I guess there would have to be a check for whether favicon.ico exists and based on that render the HTML PR #33 suggests be added.